.github/workflows/image.yaml

```diff
@@ -40,11 +40,14 @@ jobs:
         - vectorAdd
         - nbody
         - deviceQuery
+        - nvbandwidth
```

Comment:
I dont think it belongs here. It is a separate build/Dockerfile. I added another cuda-sample that should go here. See this https://github.com/NVIDIA/k8s-samples/pull/18